app/domain/sac_imports/roles/membership_entry.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module SacImports::Roles
+  class MembershipEntry
+    include EntryHelper
+
+    def initialize(row)
+      @row = row
+      @skipped = false
+    end
+
+    def errors
+      build_error_messages
+    end
```

Comment:
Ich finde `errors` ein ungünstiger name, da man sich von `ActiveModel` gewohnt ist, dass das ein Array ähnliches Ding ist.
```suggestion
    def error_messages
      build_error_messages
    end
```

Reply:
das file wird nicht mehr benötigt, von dem her hat sich das erledigt